osu.Game/Database/BeatmapDatabase.cs

```diff
@@ -112,7 +111,17 @@ public void Import(params string[] paths)
                     }
                 }
 
-                Import(new[] { beatmapSet });
+                try
+                {
+                    Import(beatmapSet);
+                }
+                catch
+                {
+                }
+                finally
+                {
+                    File.Delete(p);

```

Comment:
what if the file is in use? the file.delete will cause an unhandled exception?

common case: attempting to import a file which is in use by another application (say the web browser downloading the file).